cmd/coder/main.go

```diff
@@ -37,6 +37,8 @@ func (r *rootCmd) Subcommands() []cli.Command {
 		&shellCmd{},
 		&syncCmd{},
 		&urlsCmd{},
+		&createURLCmd{},
+		&delURLCmd{},
```

Comment:
These shouldn't be top level, they should be subcommands of `urlCmd`.

Reply:
Oops, all right. Aha, that would mean users invoke as

`$ coder urls create <env_id> <port> <access>` ?

If so, I get now why more specific command tags weren't in the proposal.